addons/ctrl-enter-post/forums.js

```diff
@@ -0,0 +1,12 @@
+export default async function ({ addon }) {
+  let type = location.pathname.split("/")[2];
+
+  let textarea = await addon.tab.waitForElement(type == "settings" ? "#id_signature" : "#id_body");
```

Comment:
i don't think you need to `waitForElement` on r2 pages

it's only needed on www pages because we load before react